shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -1326,6 +1326,14 @@ object GlutenConfig {
       .booleanConf
       .createOptional
 
+  val NATIVE_WRITE_FILES_COLUMN_METADATA_WHITE_LIST =
+    buildConf("spark.gluten.sql.native.writeFilesColumnMetadataWhiteList")
```

Comment:
"whiteList" looks misleading, as the listed items are actually removed from output. Maybe, `xxx.WriteColumnMetadataExclusionList`?

Reply:
I'm fine with this new name, changed it